ansible/ansible_collections/agentic/agent_configs/plugins/modules/deepseek_tui_mcp.py

```
@@ -0,0 +1,354 @@
+            connect_timeout=module.params.get("connect_timeout"),
+            execute_timeout=module.params.get("execute_timeout"),
+            read_timeout=module.params.get("read_timeout"),
+            enabled_tools=module.params.get("enabled_tools"),
+            disabled_tools=module.params.get("disabled_tools"),
+            extra=module.params.get("extra") or {},
+        )
+
+    existing_data = {}
+    try:
+        existing_data = load_json_file(path, default={})
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)